freertos_gdb/common.py

```diff
@@ -116,10 +125,15 @@ def __getitem__(self, idx):
 
     def __iter__(self):
         curr_node = self.head
```

Comment:
```suggestion
        curr_node = self.head
        self._index = 0
```

Could you please reinitialize index?
It seems that  `self._index` could be declared as a local variable. Such as curr_node

Reply:
done